fix: allow azure base url (#207)
empirical-run/empirical: .changeset/wild-geckos-wait.md

---
"@empiricalrun/ai": minor
---

> Allow passing a base URL for azure-openai provider to cater to more usecases

Reviewer: ```suggestion
feat: allow passing a base URL for azure-openai provider
```
